### PR TITLE
Check tournament size in backend and inform the backend if it exceeds it

### PR DESCRIPTION
**Pull request by stevebalk**
- Max player size can be configured in the pong config file
- error response message will be: "tournament size is too large"
- Changed postgres docker version back to "latest"
- Paddle start positions are now centered

### CODE REVIEW

File: backend/static/websocket_messages.yaml
Comment (StrongCoding) [inline backend/static/websocket_messages.yaml:1]: whats with tournament size is too small?

File: backend/pong/tournament/tournament_session_handler.py
Comment (StrongCoding) [inline backend/pong/tournament/tournament_session_handler.py:1]: is 2 player tournament useful?

File: backend/static/websocket_messages.yaml
Comment (stevebalk) [reply backend/static/websocket_messages.yaml:1]: Joa, das sollte man auch returnen

File: backend/pong/tournament/tournament_session_handler.py
Comment (stevebalk) [reply backend/pong/tournament/tournament_session_handler.py:1]: https://giphy.com/gifs/paramountplus-nickelodeon-icarly-episode303-o8JxccZ9VdMlqsA3kX

**Review by StrongCoding — APPROVED** [review]

**Pull request merged**